refarch-backend/pom.xml

```diff
@@ -439,6 +454,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!-- Flyway Maven plugin for local database management
+                     Usage:
+                     - Clean database: mvn flyway:clean
+                     - Apply migrations: mvn flyway:migrate
+                     - Reset and migrate: mvn flyway:clean flyway:migrate
+                     
+                     Note: Use -Dflyway.cleanDisabled=false when running clean command
```

Comment:
Why not set cleanDisabled to false by default? Is this too risky? I think when I execute the clean goal I know what I'm doing.

Reply:
I was using the flyway-maven-plugin to do other stuff (like baselining a prod db to reduce hundreds of migrations to 1) in the past. So therefore I think it's safer not to set this configuration, because accidentally running clean on prod can make a busy day ;)

Reply:
Ok fine for me, lets get this thing merged ;)